test/regression/standalone/directiveSpec.js

```diff
@@ -28,7 +28,7 @@ module.exports = function() {
     assert.ifResourceNameIsPresentAt('/resource2', 4);
   });
 
-  it('should be able to display security schemes', function () {
+  xit('should be able to display security schemes', function () {
```

Comment:
Could you add some TODO comments here explaining why you have to skip these tests?